localstack/utils/common.py

```diff
@@ -118,11 +119,15 @@ def filter_line(line):
             """ Return True if this line should be filtered, i.e., not printed """
             return '(Press CTRL+C to quit)' in line
 
+        outfile_orig = self.outfile
+        outfile = self.outfile or os.devnull
+        if self.log_listener and outfile in [None, os.devnull]:
```

Comment:
Python noob here. Can `outfile` be `None`? It will fallback to `os.devnull` when it's `None`, right?

Reply:
Great catch, thanks @hisener 👍 - updated the PR to fix this.